Code/Tools/ProjectManager/Source/Application.h

```diff
@@ -22,6 +22,8 @@ namespace AZ
 
 namespace O3DE::ProjectManager
 {
+    void SetupCrashHandler();
+
```

Comment:
Another possibility to avoid adding those extra PAL .cpp files for Mac and Linux would be to make a trait (e.g. AZ_TRAIT_PROJECT_MANAGER_CRASH_HANDLER) defined in the existing `Platform/Windows/ProjectManager_Traits_Windows.h` files and then just wrap the function definition and call with that #ifdef like so.

```suggestion
#if AZ_TRAIT_PROJECT_MANAGER_CRASH_HANDLER
    void SetupCrashHandler();
#endif
```

Reply:
Good idea, I'll do that!